devtools/gradle/src/main/java/io/quarkus/gradle/builder/QuarkusModelBuilder.java

```diff
@@ -331,9 +337,14 @@ private void collectDependencies(ResolvedConfiguration configuration,
             final DependencyImpl dep = initDependency(a);
             if (LaunchMode.DEVELOPMENT.equals(mode) &&
                     a.getId().getComponentIdentifier() instanceof ProjectComponentIdentifier) {
-                Project projectDep = project.getRootProject()
-                        .findProject(((ProjectComponentIdentifier) a.getId().getComponentIdentifier()).getProjectPath());
-                addDevModePaths(dep, a, projectDep);
+                Optional<IncludedBuild> includedBuild = includedBuild(project, a.getName());
```

Comment:
Couldn't this be replaced with `project.getGradle().includedBuild(name)`?

Reply:
yes, I just wrapped that in a `try/catch` because `project.getGradle().includedBuild(name)` throws an exception if the project is not an included build.